### PR TITLE
Move from "Experimental" to "Extensions"

### PR DESCRIPTION
**Pull request by caesarxuchao**
ref #15378 #15409

### CODE REVIEW

**Comment by lavalamp** [comment]
Docs need updating, looks like.

**Comment by caesarxuchao** [comment]
e2e is flake:

```
Expected error:
    <*errors.errorString | 0xc2080fc320>: {
        s: "gave up waiting for pod 'redis-master-i7ovh' to be 'running' after 5m0s",
    }
    gave up waiting for pod 'redis-master-i7ovh' to be 'running' after 5m0s
not to have occurred
```

**Comment by lavalamp** [comment]
Tests still not very happy?

**Comment by caesarxuchao** [comment]
@lavalamp  this one is ready

File: docs/proposals/api-group.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Documentation for other releases can be found at
 
 2. Spelling the URL:
 
-  The URL is in the form of prefix/group/version/. The prefix is hard-coded in the client/unversioned.Config (see [here](../../pkg/client/unversioned/experimental.go#L101)). The client should be able to figure out `group` and `version` using the RESTMapper. For a third-party client which does not have access to the RESTMapper, it should discover the mapping of `group`, `version` and `kind` by querying the server as described in point 2 of #server-side-implementation.
+  The URL is in the form of prefix/group/version/. The prefix is hard-coded in the client/unversioned.Config. The client should be able to figure out `group` and `version` using the RESTMapper. For a third-party client which does not have access to the RESTMapper, it should discover the mapping of `group`, `version` and `kind` by querying the server as described in point 2 of #server-side-implementation.
```
Comment (lavalamp) [inline docs/proposals/api-group.md]: This seems like a wrong change?

**Comment by lavalamp** [comment]
one last comment + needs rebase

File: docs/proposals/api-group.md (hunk 1 above)
Comment (caesarxuchao) [inline docs/proposals/api-group.md]: I just removed the code reference because the code is being rapidly changed.

**Comment by caesarxuchao** [comment]
rebased.

File: docs/proposals/api-group.md (hunk 1 above)
Comment (lavalamp) [inline docs/proposals/api-group.md]: OK

**Comment by lavalamp** [comment]
LGTM-- please cherrypick

**Pull request merged**